test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -66,6 +63,36 @@ uint64_t clz(uint64_t _v)
 	return r;
 }
 
+/// Count trailing zeros for uint64
+uint64_t ctz(uint64_t _v)
+{
+	uint64_t r = 0;
+	unsigned count = 64;
+	while (count-- && (_v & 1))
+	{
+		r++;
+		_v >>= 1;
+	}
+	return r;
+}
```

Comment:
Does this work? Let's skip talking about ``_v = 0``, but e.g. for ``_v = (1 << 32);``... this is supposed to return ``32``, right? But ``_v & 1`` will be false in the first iteration, so it *will* return zero...
Maybe rather:
```
uint64_t ctz(uint64_t _v)
{
  if (!_v) return 64;
  uint64_t r = 0;
  while (!(_v & 1)) { r++; v >>= 1; }
  return r;
}
```